mappings/net/minecraft/enchantment/Enchantment.mapping

```diff
@@ -1,12 +1,32 @@
 CLASS net/minecraft/unmapped/C_jxtrubuh net/minecraft/enchantment/Enchantment
+	FIELD f_cvunyugc HOLDER_CODEC Lcom/mojang/serialization/Codec;
```

Comment:
```suggestion
	FIELD f_cvunyugc ENTRY_CODEC Lcom/mojang/serialization/Codec;
```
this is the mapping I use in #598, as when you say holder it's not clear what exactly it's holding